@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -732,6 +732,19 @@ export interface SolidLineTechnique extends BaseTechnique, PolygonalTechnique {
      * Clip the line outside the tile if `true`.
      */
     clipping?: boolean;
+    /**
+     * Color of a line's outline in hexadecimal or CSS-style notation, for example: `"#e4e9ec"`,
+     * `"#fff"`, `"rgb(255, 0, 0)"`, or `"hsl(35, 11%, 88%)"`.
+     */
+    outlineColor?: string;
+    /**
+     * Width of a line's outline in `metricUnit`s for different zoom levels.
+     */
+    outlineWidth?: number;
```

Comment:
From this doc alone, it may me not clear if `outlineWidth` is 'width' of geometry that draws outline or is it `width` added around main part of line.

So, strictly speaking if line with with `B` has outline with width `B`, then total visible width should be `A+B+B` (i.e main line +2 outlines from both sides).
This way of defining property reveals impl detail that outline is independent line with independent width.

So, either clarify doc, or change PR, so `outlineWidth` a _pure outline_ width.

Reply:
Will change the prefix of these properties to ``secondary``, since actually ``outline`` doesn't really define the behaviour of the second geometry as you pointed out.